gemini/src/main/java/com/techempower/cache/EntityStore.java

```diff
@@ -1759,6 +1777,14 @@ public void clearCachedResponses()
     return Collections.unmodifiableList(typeAdapters);
   }
 
+  /**
+   * Returns whether this EntityStore has completed initialization.
+   */
+  public boolean isInitialized()
```

Comment:
Useful so the CacheMessageManager can ignore messages until the store is initialized.